.env.example

```
@@ -1,3 +1,5 @@
 DISCORD_TOKEN=
 MAIN_CHANNEL_ID=
 GUILD_ID=
+PREFIX=
+FEATURE=
```

これらはデフォルト値が存在するので, 例もそれに合わせたほうがいいと思います.
```suggestion
PREFIX=!
FEATURE=MESSAGE_CREATE,MESSAGE_UPDATE,COMMAND,VOICE_ROOM,ROLE,EMOJI
```